lib/modem_info/modem_info.c

```diff
@@ -292,7 +371,9 @@ static int modem_info_parse(const struct modem_info_data *modem_data, char *buf)
 
 enum at_param_type modem_info_type_get(enum modem_info info)
 {
-	__ASSERT(info < MODEM_INFO_COUNT, "Invalid argument.");
+	if (info >= MODEM_INFO_COUNT) {
```

Comment:
`info` could be called something more descriptive. `info_type` perhaps?

Reply:
OK